ocpp-v1_6/src/test/java/eu/chargetime/ocpp/wss/BaseWssSocketBuilderTest.java

```diff
@@ -0,0 +1,119 @@
+package eu.chargetime.ocpp.wss;
+
+import org.hamcrest.CoreMatchers;
+import org.junit.Test;
+
+import javax.net.ssl.SSLSocketFactory;
+
+import java.io.IOException;
+import java.net.*;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assert.*;
+import static org.mockito.Matchers.any;
+import static org.mockito.Mockito.*;
+
+/**
+ * Test for {@link BaseWssFactoryBuilder}
+ */
+public class BaseWssSocketBuilderTest {
+
+    @Test
+    public void builder() {
+        BaseWssSocketBuilder builder = BaseWssSocketBuilder.builder();
+        assertThat(builder, is(CoreMatchers.any(BaseWssSocketBuilder.class)));
+    }
+
+
+    @Test
+    public void build() throws IOException, URISyntaxException {
+        SSLSocketFactory sslSocketFactory = mock(SSLSocketFactory.class);
+        Socket expectedSocket = mock(Socket.class);
+        Socket socket = mock(Socket.class);
+        when(socket.isBound()).thenReturn(false);
+        when(sslSocketFactory.createSocket(socket,"fake", 101, false)).thenReturn(expectedSocket);
```

Comment:
Please, put space after comma. There and in other places.

Reply:
Done for the particular case. In general I would propose to define reference for the project for a particular codestyle so it could be formatted automatically in accrodance (e.g. https://google.github.io/styleguide/javaguide.html)